server/server/models/rider.js

```diff
@@ -0,0 +1,32 @@
+const mongoose = require('mongoose');
+
+const RiderSchema = new mongoose.Schema({
+    user: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'User',
+        required: true
+    },
+    license: {
+        type: String,
+        required: true,
+        unique: true
+    },
+    nationalityId: {
+        type: String,
+        required: true
```

Comment:
could we add unique property to nationalityId

Reply:
@Dana-MO definitely, on it